### PR TITLE
fix duplicate url set call in table converter

### PR DESCRIPTION
**Pull request by alexreid-db**
**PR Checklist**

- [x] A description of the changes is added to the description of this PR.
- [ ] If there is a related issue, make sure it is linked to this PR.
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added or modified a feature, documentation in `docs` is updated

**Description of changes**

Removed duplicate (and non null checking) url set call in table converter code. Also, cleaned up some unused imports and exceptions thrown.

### CODE REVIEW

**Review by Fokko — APPROVED** [review]
LGTM!

File: server/src/main/java/io/unitycatalog/server/persist/converters/TableInfoConverter.java [hunk 1]
```diff
@@ -94,7 +92,6 @@ public static TableInfoDAO convertToDAO(TableInfo dto) {
                 .url(dto.getStorageLocation() != null ? dto.getStorageLocation() : null)
                 .type(dto.getTableType().toString())
                 .dataSourceFormat(dto.getDataSourceFormat().toString())
-                .url(dto.getStorageLocation())
```
Comment (rameshchandra) [inline server/src/main/java/io/unitycatalog/server/persist/converters/TableInfoConverter.java:97]: @alexreid-db Isn't `dto.getStorageLocation() != null ? dto.getStorageLocation() : null` equivalent to `dto.getStorageLocation()`? So, shouldn't this line be left as is and line 94 be removed?

**Review by rameshchandra — CHANGES_REQUESTED** [review]
Thank you for the fix.

File: server/src/main/java/io/unitycatalog/server/persist/converters/TableInfoConverter.java (hunk 1 above)
Comment (alexreid-db) [reply server/src/main/java/io/unitycatalog/server/persist/converters/TableInfoConverter.java:97]: yeah, sorry, didn't look at the line above closely enough

**Review by rameshchandra — APPROVED** [review]
LGTM

**Pull request merged**